src/app/components/events/overview1/overview1.component.ts

```diff
@@ -21,7 +21,7 @@ export class Overview1Component {
         let event: AEvent;
 
         for (let i = 0; i < RANDOM_GENERATED_EVENTS; i++) {
-            event = new AEvent();
+            event = new AEvent(i + 1);
 
             event.title = `The Fantastic event-${i}`;
```

Comment:
This title is also de-synchronized..

Reply:
Yes as intended. But I've added the '+ 1' to the title now as well :)